compareECLFiles.cmake

```diff
@@ -37,9 +37,38 @@ function(add_test_runSimulator)
                            -f ${PARAM_FILENAME}
                            -n ${PARAM_PROCS}
                TEST_ARGS ${TEST_ARGS}
-               CONFIGURATION extra)
+               CONFIGURATION ${PARAM_CONFIGURATION})
```

Comment:
since we are using this for 'normal' tests now, please add
```
set_test_properties(runsimulator/${PARAM_CASENAME} PROPERTIES PROCESSORS ${PARAM_PROCS})
```